tests/test_add_new_employee.py

```diff
@@ -0,0 +1,73 @@
+"""
+Test for /survey/admin/new_employee endpoint.
+The test is used to add new employee to the database
+The following test is covered:
+1. Add a new employee
+2. Make sure employee has been added by getting the employee details via /survey/admin/get_employee_by_email endpoint
+3. The employee ID of the newly added employee is <Previous_id + 1>
+4. If empty data is passed to the endpoint to /survey/admin/new_employee, make sure we get error message
+"""
+
+import os
+import json
+import requests
+import pytest
+from datetime import datetime , timedelta, date
+from dateutil.relativedelta import relativedelta, FR
```

Comment:
Do we need line 15, 16  we are not using these libraries?

Reply:
Hey @mohanqxf2 , you're right, those libraries are not needed , have removed them and updated the PR